mozetl/topline/topline_dashboard.py

```diff
@@ -0,0 +1,197 @@
+""" Reformat Data for Topline Dashboard
+
+This script replaces `run.sh` and `v4_reformat.py` that generates data ready
+for dashboard consumption. The original system used a pipeline consisting of a
```

Comment:
Let's add a link to the dashboard @ `http://metrics.services.mozilla.com/firefox-dashboard/` in this comment and call out that the old data files were located in `data/v4-weekly.csv` and `data/v4-monthly.csv`. That might be helpful if we need to trace the data's origin in the future.